src/core/inventory.c

```diff
@@ -336,4 +346,21 @@ mender_inventory_work_function(void) {
     return ret;
 }
 
+static mender_err_t
+artifact_name_device_type_cb(mender_keystore_t **inventory, uint8_t *inventory_len) {
+    mender_err_t             ret;
+    static mender_keystore_t basic_device_info[] = {
```

Comment:
So persistent inventory is expected to be statically allocated, and will never be free'd by a call to `mender_utils_keystore_delete()`?